src/framework/theme/components/popover/helpers/trigger.helper.ts

```diff
@@ -32,7 +25,7 @@ const NB_TRIGGERS = {
    * */
   [NbPopoverMode.CLICK](host: HTMLElement, getContainer: Function, document: Document): NbPopoverTrigger {
     return {
-      open: observableEmpty(),
+      open: EMPTY$,
```

Comment:
Why do we need $ postfix?

Reply:
because it's observable, otherwise it's not obvious